### PR TITLE
Fix compatibility with PHP-8

### PR DESCRIPTION
**Pull request by bp-rest**
With PHP-7.x missing arginfo has been triggering warnings:
  NOTICE: PHP message: PHP Warning:  Missing arginfo for radius_auth_open() in Unknown on line 0
With PHP-8 this has become mandatory.

In contrast to pull request #6 this fully defines the arguments rather than using basic empty placeholder arginfo.

### CODE REVIEW

**Comment by rcmcdonald91** [comment]
@LawnGnome I reached out to you via email concerning PHP8 support. Let me know if you are interested in continuing maintenance of this extension. Otherwise, I am willing to takeover.

Best,
CM

File: radius.c [hunk 1]
```diff
@@ -61,39 +61,199 @@ ZEND_DECLARE_MODULE_GLOBALS(radius)
 /* True global resources - no need for thread safety here */
 static int le_radius;
 
+/* {{{ arginfo */
+ZEND_BEGIN_ARG_INFO_EX(arginfo_radius_auth_open, 0, 0, 0)
```
Comment (kocsismate) [inline radius.c:65]: I'd suggest that you should use gen_stub.php (in the build directory) for defining arginfo structures. I recently created a similar PR for the scrypt extension: https://github.com/DomBlack/php-scrypt/pull/63

Comment (rcmcdonald91) [reply radius.c:65]: Yes. I am already there and nearly complete with a rebase against the latest (8.1) skeleton.

**Comment by dmitryluhtionov** [comment]
Any chance of a commit?

**Comment by mbretter** [comment]
I am willing to merge this PR, but I have not worked for almost 20yrs on this extension and have not the time to test this.
On the other hand, this packages becomes more and more useless without PHP8 support.

**Comment by dmitryluhtionov** [comment]
You can transfer the rights to maintenance to `bp-rest`

пн, 12 дек. 2022 г. в 15:58, Michael Bretterklieber <
***@***.***>:

**Comment by mbretter** [comment]
> You can transfer the rights to maintenance to `bp-rest` 

I would, but I do not have the permissions ...

**Comment by dmitryluhtionov** [comment]
Thanks.
Can you create new release on pecl.php.net ?

пн, 12 дек. 2022 г. в 16:18, Michael Bretterklieber <
***@***.***>:

**Comment by dmitryluhtionov** [comment]
Can you copy radius-coa.php
<https://github.com/gblanchard/pecl-radius/blob/master/examples/radius-coa.php>
and radius-disconnect.php
<https://github.com/gblanchard/pecl-radius/blob/master/examples/radius-disconnect.php>
from https://github.com/gblanchard/pecl-radius/tree/master/examples
repository ?

And, partially,
https://github.com/gblanchard/pecl-radius/blob/master/radius_init_const.h
and more files from this commit:
https://github.com/gblanchard/pecl-radius/commit/60fa4aa4416df03e23b978190ceaf1faa4e1853e


пн, 12 дек. 2022 г. в 17:07, Dmitry Luhtionov ***@***.***>:

**Pull request merged**